target/linux/ramips/image/mt76x8.mk

```diff
@@ -202,13 +202,22 @@ define Device/wl-wn575a3
 endef
 TARGET_DEVICES += wl-wn575a3
 
-define Device/widora-neo
-  DTS := WIDORA-NEO
+define Device/widora-neo-16M
+  DTS := WIDORA-NEO-16M
   IMAGE_SIZE := $(ralink_default_fw_size_16M)
-  DEVICE_TITLE := Widora-NEO
+  DEVICE_TITLE := Widora-NEO-16M
+  SUPPORTED_DEVICES := Widora-NEO widora-neo WIDORA-NEO Widora widora
```

Comment:
```
SUPPORTED_DEVICES += widora-neo
```

only.

Please do the test on your own. Install a 17.01 widora-neo neo image and try to update to the widora-neo-16M image. Try to update from a widora-neo-16M  image to another widora-neo-16M image.

Do a similar test for the 32M image and check what doesn't work any longer.

Reply:
ok thx. Some customer firmwares named like that so I wrote something else there.

But the ``SUPPORTED_DEVICES += widora-neo Widora`` as I think the official image named it as "Widora", which indicates here: https://github.com/huming2207/openwrt-pandorabot/blob/master/target/linux/ramips/image/Makefile#L1031  
I'm not quite sure about that as I haven't deal with OpenWrt 15.05 quite much before.

Reply:
It isn't about the name used by any official firmware.

Please open the sysupgrade.bin in a hex editor and scroll to the end. The boardname from `/tmp/sysinfo/board_name` is compared with the supported_devices entry from the image metadata at the end of your image (sysupgrade.bin).